lib/jekyll/converters/markdown/redcarpet_parser.rb

```diff
@@ -91,11 +87,29 @@ def class_with_proper_highlighter(highlighter)
         end
 
         def convert(content)
-          @redcarpet_extensions[:fenced_code_blocks] = !@redcarpet_extensions[:no_fenced_code_blocks]
-          @renderer.send :include, Redcarpet::Render::SmartyPants if @redcarpet_extensions[:smart]
-          markdown = Redcarpet::Markdown.new(@renderer.new(@redcarpet_extensions), @redcarpet_extensions)
+          @redcarpet_extensions[:fenced_code_blocks] = \
+            !@redcarpet_extensions[:no_fenced_code_blocks]
```

Comment:
What happens if we shorten this class like previously discussed and do something like:

```
module Parent::NameSpace::Blah
  class NameSpace
  end
end
```

It seems like something that short really shouldn't be split off.  That or we could increase the limit again.  Seems like this limit is constantly sabotaging readability.